packages/eslint-plugin/src/rules/no-context-default-value/index.js

```diff
@@ -40,9 +34,8 @@ module.exports = createRule({
       },
     ],
   },
-  defaultOptions: [],
+  defaultOptions: /** @type {Options[]} */ ([]),
```

Comment:
in order to provide proper rule type for test, `defaultOptions` need to be typed to rule option.